lib/reporters/min.js

```diff
@@ -8,6 +8,9 @@
 
 var Base = require('./base');
```

Comment:
This is a question, not request for change:
As _all_ reporters besides `html` are Node-based, are they required to retain ES5-compatiblity?

Reply:
I think, simply because they're all pulled in to the bundle.  *Not* pulling them in to would break people using certain headless browser setups.

Ideally we shouldn't have to manage the language level on a file-by-file or directory-by-directory basis, and would have a transpilation step.  I don't want to have to think about it, and neither should our contributors.

Reply:
(yes, to pile on w/ the confusion, the node.js-based reporters *can and do* run in a browser on the command-line)

Reply:
Wish I could say I understand what you just wrote, but I don't. Many are terminal-based (e.g., Nyan) and _should_ fail if attempted to run in browser. I was about to add more error checking which would cause `Error` to be thrown if terminal cursor positioning was unsupported in a new PR. You saying that won't work due to browser?

Reply:
See what [mochify](https://npm.im/mochify) does.  Essentially it takes Mocha and pipes it thru a headless browser, but it looks like it's running as it would with Node.js.  That means you can use console-based reporters with it.

It's unclear to me if what you're doing would cause a problem.  My suggestion is to download Mochify and `npm link` it against your `mocha` working copy & see what happens.